tests/architecture/test_components.py

```
@@ -468,72 +469,41 @@ def setup_method(self):
+    def test_builder_integration(self):
+        """Test integration of current PromptixBuilder class with real workspace."""
+        # Test with an existing prompt (SimpleChat should exist)
+        try:
+            builder = Promptix.builder("SimpleChat")
+            # Test that builder exists and is functional
+            assert hasattr(builder, 'build')
+            assert hasattr(builder, 'with_user_name')
+            
+            # Try to build a basic config
+            config = (builder
+                     .with_user_name("TestUser")
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)